packages/core/src/components/Table/index.tsx

```diff
@@ -20,6 +20,8 @@ export type Props = {
   loading?: boolean;
   /** Applies verticalAlign: middle to child cells. */
   middleAlign?: boolean;
+  /** Disable responsive wrapper. */
+  noWrap?: boolean;
```

Comment:
a story with `noWrap` and rows with MenuToggle in it to demonstrate these two working together, would be great!

Reply:
PTAL!